core/templates/components/button-directives/social-buttons.component.ts

```diff
@@ -16,21 +16,24 @@
  * @fileoverview Directive for the social buttons displayed in footer.
  */
 
-require('domain/utilities/url-interpolation.service.ts');
+import { Component } from '@angular/core';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-angular.module('oppia').directive('socialButtons', [
-  'UrlInterpolationService', function(UrlInterpolationService) {
-    return {
-      restrict: 'E',
-      scope: {},
-      bindToController: {},
-      template: require('./social-buttons.directive.html'),
-      controllerAs: '$ctrl',
-      controller: [function() {
-        var ctrl = this;
-        ctrl.getStaticImageUrl = function(imagePath) {
-          return UrlInterpolationService.getStaticImageUrl(imagePath);
-        };
-      }]
-    };
-  }]);
+@Component({
+  selector: 'social-buttons',
+  templateUrl: './social-buttons.directive.html',
```

Comment:
Can you rename the html file to social-buttons.component.html

Reply:
Done